src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -49,8 +49,7 @@ static CompletionItemRules MakeRule((bool importDirective, bool preselect, bool
             {
                 // '<' should not filter the completion list, even though it's in generic items like IList<>
                 var generalBaseline = CompletionItemRules.Default.
-                    WithFilterCharacterRule(CharacterSetModificationRule.Create(CharacterSetModificationKind.Remove, '<')).
-                    WithCommitCharacterRule(CharacterSetModificationRule.Create(CharacterSetModificationKind.Add, '<'));
```

Comment:
`<` is part of the default commit characters set and doesn't need to be specified here as best I can tell, cc @genlu